src/html/partials/key-factors.html

```diff
@@ -1,4 +1,37 @@
 <section class="factors-section">
+  
   <h2 class="sm-title">Why People Don't Adhere</h2>
   <p class="bg-paragraph">A myriad of factors conspire to prevent people from sticking to their treatment programs. But a few are most telling.</p>
+
+  <div class="factors-section__human-figure-wrapper">
+    <div class="factors-section__human-figure-gear"></div>
+    <div class="factors-section__human-figure-heart"></div>
+    <div class="factors-section__human-figure-dollar"></div>
+    <div class="factors-section__human-figure-tally"></div>
+  </div>
```

Comment:
Might be a bit premature to add empty divs for things, but I see where you're going, so: 👍 
Also, I'm assuming these are the extra icons that only show on the mobile size?

Reply:
Gotcha, wasn't sure if that was good to add now or later when getting into the weeds with styling. And yes, these are for mobile.

Reply:
Will comment out for now